src/server/worker/worker.go

```diff
@@ -74,6 +75,9 @@ func NewWorker(
 
 	worker.APIServer = server.NewAPIServer(driver, worker.status, env.Config().PodName)
 
+	span, _ := driver.AddSpanToAnyPipelineTrace("/worker.Master/Startup")
+	tracing.FinishAnySpan(span)
```

Comment:
My guess is that the goal of this is to try and deduce whether a slow job is blocking on worker startup (i.e. datums aren't being claimed due to scheduling delays)—is that right? Having a length-0 span mark an event strikes me as a somewhat unusual (but very creative!) use of tracing, and I wonder if it would be possible to have a span measure "worker startup time" (i.e. time from startup to first datum) which I think would give at least as much information and seems more conventional.

WDYT? We could do this in a followup PR too.